### PR TITLE
Patch issue with sphinx>3/docutls>0.15

### PR DESCRIPTION
**Pull request by melund**
This patches an issue with the html-toggle element in the theme and docutils>0.15 (sphinx > 3.5). 

We need to patch this because this theme is no longer maintained. 

Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by melund** [comment]
@conda-forge-admin, please rerender

**Comment by melund** [comment]
@jakirkham, this is ready to be merged. It is a very minor patch to this otherwise abandoned sphinx theme. The patch ensure ["html-toggle"](https://cloud-sptheme.readthedocs.io/en/latest/cloud_theme_test.html#toggleable-section) feature, also works with sphinx > 3.

**Comment by melund** [comment]
@jakirkham, feel free to also add me as a user of this feedstock if you like.

**Comment by melund** [comment]
@jakirkham, just friendly ping :)

**Comment by melund** [comment]
@conda-forge/help-python : Could you help getting this reviewed/merged? (Also, feel free to add me as the maintainer of the feedstock)

File: recipe/meta.yaml [hunk 1]
```diff
@@ -11,10 +11,12 @@ source:
   patches:
     - 0001-patch-sphinx-4-support.patch
     - 0002-patch-jinja-markup-deprecation.patch
+    - 0003-fix-pycompat-warning.patch
+    - 0004-fix-docutils-0.17-issues-with-toggle-html.patch
 
 build:
   noarch: python
-  number: 2
+  number: 4
```
Comment (jakirkham) [inline recipe/meta.yaml]: Is there a reason 3 is skipped?

File: recipe/meta.yaml [hunk 2]
```diff
@@ -11,10 +11,12 @@ source:
   patches:
     - 0001-patch-sphinx-4-support.patch
     - 0002-patch-jinja-markup-deprecation.patch
+    - 0003-fix-pycompat-warning.patch
+    - 0004-fix-docutils-0.17-issues-with-toggle-html.patch
```
Comment (jakirkham) [inline recipe/meta.yaml:15]: Are there upstream PRs for these?

**Comment by jakirkham** [comment]
Thanks Morten! 🙏

Sorry for the delay

Have asked a couple questions below

**Comment by melund** [comment]
@jakirkham 


Thanks Morten! 🙏

> Sorry for the delay
> Have asked a couple questions below

No problem. I can't see the questions though. Maybe you forgot to submit ...

**Comment by jakirkham** [comment]
Weird thought I had wrote that as a review comment. Sorry about that. They should now render above

File: recipe/meta.yaml (hunk 1 above)
Comment (melund) [reply recipe/meta.yaml]: Good catch. I had a local intermediate version which was 3. I will change this.

File: recipe/meta.yaml (hunk 2 above)
Comment (melund) [reply recipe/meta.yaml:15]: No. The project was moved from bitbucket to foss.heptapod.net just before it was abandoned. The project is misconfigured to not allow forks, so we [can't contribute PRs](https://foss.heptapod.net/doc-utils/cloud_sptheme/-/issues/45#note_158949).

There are issues reporting these things:
https://foss.heptapod.net/doc-utils/cloud_sptheme/-/issues

**Comment by melund** [comment]
@jakirkham; Thanks, I tried to address the your issues and questions.

**Comment by jakirkham** [comment]
@conda-forge-admin, please re-render

**Comment by jakirkham** [comment]
Thanks Morten! 🙏

**Pull request merged**